### PR TITLE
Added option to check current package version

### LINKED ISSUE #25: Add  a -V/--version option
It should just output the version and quit.

### PR DESCRIPTION
**Pull request by shivamshan**
This PR makes the following changes:-

- adds command line option to check current python version
- black formatted pyempaq/main.py

Thus, this PR closes #25.

## Code Proof
```
>>pyempaq -V
pyempaq 0.2.2

>>pyempaq --version
pyempaq 0.2.2
```

Signed-off-by: Shivam Shandilya <shivam.stpaulsdarjeeling@gmail.com>

### CODE REVIEW

File: pyempaq/main.py [hunk 1]
```diff
@@ -15,6 +15,7 @@
 import venv
 import zipapp
 from collections import namedtuple
+import pkg_resources
```
Comment (shivamshan) [inline pyempaq/main.py:18]: importing pkg_resources module that will fetch the version

File: pyempaq/main.py [hunk 2]
```diff
@@ -144,8 +151,17 @@ def main():
     """Manage CLI interaction and call pack."""
     parser = argparse.ArgumentParser()
     parser.add_argument(
-        "source", type=pathlib.Path,
-        help="The source file (pyempaq.yaml) or the directory where to find it.")
+        "source",
+        type=pathlib.Path,
+        help="The source file (pyempaq.yaml) or the directory where to find it.",
```
Comment (shivamshan) [inline pyempaq/main.py:156]: formatted by running black

File: pyempaq/main.py [hunk 3]
```diff
@@ -47,7 +50,11 @@ def logged_exec(cmd):
     print(f"Executing external command: {cmd}")
     try:
         proc = subprocess.Popen(
-            cmd, stdout=subprocess.PIPE, stderr=subprocess.STDOUT, universal_newlines=True)
+            cmd,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.STDOUT,
+            universal_newlines=True,
+        )
```
Comment (shivamshan) [inline pyempaq/main.py:57]: formatted by black

File: pyempaq/main.py [hunk 4]
```diff
@@ -38,7 +39,9 @@ def find_venv_bin(basedir, exec_base):
         # windows environment
         return bin_dir / f"{exec_base}.exe"
 
-    raise RuntimeError(f"Binary not found inside venv; subdirs: {list(basedir.iterdir())}")
+    raise RuntimeError(
+        f"Binary not found inside venv; subdirs: {list(basedir.iterdir())}"
+    )
```
Comment (shivamshan) [inline pyempaq/main.py:44]: formatted by black

File: pyempaq/main.py [hunk 5]
```diff
@@ -144,8 +151,17 @@ def main():
...
+        type=pathlib.Path,
+        help="The source file (pyempaq.yaml) or the directory where to find it.",
+    )
+    parser.add_argument(
+        "--version",
+        "-V",
+        action="version",
+        version=f"%(prog)s {pkg_resources.get_distribution('pyempaq').version}",
+        help="The current version of pyempaq.",
+    )
```
Comment (shivamshan) [inline pyempaq/main.py:164]: added **--version** and **-V** option that will print the version of the module and exit

File: pyempaq/main.py [hunk 6]
```diff
@@ -144,8 +151,17 @@ def main():
...
-        help="The source file (pyempaq.yaml) or the directory where to find it.")
+        "source",
+        type=pathlib.Path,
+        help="The source file (pyempaq.yaml) or the directory where to find it.",
+    )
+    parser.add_argument(
+        "--version",
+        "-V",
+        action="version",
+        version=f"%(prog)s {pkg_resources.get_distribution('pyempaq').version}",
```
Comment (facundobatista) [inline pyempaq/main.py:162]: While this works when `pyempaq` is installed, it doesn't if you run it from the sources like `python -m pyempaq`. It needs to cover that case.

Also, please use a separate TESTED function. Thanks!

**Review by facundobatista — CHANGES_REQUESTED** [review]
Hello! Thanks for this! I annotated a detail inside.

Beyond that, though, this general request: do not run black on the project, thanks!

**Comment by facundobatista** [comment]
Done n #25.

**Pull request closed without merging**